tests/utils/test_modeling_tf_core.py

```diff
@@ -67,7 +67,6 @@
 class TFCoreModelTesterMixin:
     model_tester = None
     all_model_classes = ()
-    all_generative_model_classes = ()
```

Comment:
why we don't need `def all_generative_model_classes` for TF ..?

Reply:
incorrectly moved 

(I tried to apply the same pattern on TF, but it has too many broken models. Then I reverted the changes. This one was incorrectly reverted, I'm going to chase down the TF diff to 0)